ui/src/components/icon/QIcon.js

```diff
@@ -60,8 +62,8 @@ export default Vue.extend({
         }
       }
 
-      if (icon.startsWith('M') === true) {
-        const [ def, viewBox ] = icon.split('|')
+      if (icon.startsWith('M') === true || icon.startsWith('m') === true) {
```

Comment:
An svg starting with "m" is kind of pointless ("m" is for "move to position relative to current one", where current one is always 0,0). Let's try to make all of them start with "M" to avoid one more "if" statement. QIcon needs to be rendered as fast as possible as its probably the most used one in the entire framework.

Reply:
I was more concerned about people using custom. A couple of the test icon packages I was using had lower-case 'm' and had issues (wouldn't render). That because the path was grabbed without change. I did modify that in the build code, so I am fine reverting this back to just 'M'.